website/integrations/services/google/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Google Workspace
+---
+
+<span class="badge badge--primary">Support level: authentik</span>
+
+## What is Google Workspace
+
+From https://en.wikipedia.org/wiki/Google_Workspace
+
+:::note
+Google Workspace is a collection of cloud computing, productivity and collaboration tools, software and products developed and marketed by Google.
```

Comment:
I'll keep this as-is since it's a quote from wikipedia

....Actually by that logic we should migrate from `:::info` for this to the blockquote `> ` syntax